features/admin/quota.feature

```diff
@@ -572,7 +572,7 @@ Feature: Quota related scenarios
       | pods\\s+0\\s+2                   |
       | replicationcontrollers\\s+0\\s+3 |
       | resourcequotas\\s+1\\s+3         |
-      | secrets\\s+9\\s+5                |
+      | secrets\\s+6\\s+5                |
```

Comment:
Will it works for older versions, e.g, 4.6, 4.7 ?

Reply:
@liangxia we do not plan to run the case on lower versoin of ocp

Reply:
Then please remove related versions from Polarion, and from above tags.

Reply:
Thanks, version updated on both Poalrion and tags